tests/test_src/test_provider/test_proxyprovider.cpp

```diff
@@ -144,4 +81,181 @@ TEST(ProxyProvider, getAllESDTokenBalances_multipleTokens)
     EXPECT_FALSE(esdts.at("0009O-8742a4").empty());
 }
 
+class GenericProxyProviderTxFixture : public ::testing::Test
```

Comment:
👍 Nice refactoring for integration tests.

Reply:
Thank you ! :smile: